packages/omg-js-rootchain/src/rootchain.js

```diff
@@ -174,25 +194,27 @@ class RootChain {
   }
 
   /**
-   * Processes any exits that have completed the challenge period.
-   * @method processExits
-   * @param {string} token Address of the token to process.
-   * @param {number} topUtxoPos First exit that should be processed. Set to zero to skip the check.
-   * @param {number} exitsToProcess Maximum number of exits to process.
+   * Processes any exit that has completed the challenge period.
+   * @method processExit
+   * @param {string} exitId An exit id returned from startStandardExit.
+   * @param {string} token An address of the token to exit.
    * @param {Object} txOptions transaction options, such as `from`, gas` and `privateKey`
    * @return {string} transaction hash of the call
    */
-  async processExits (token, topUtxoPos, exitsToProcess, txOptions) {
+  async processExit (exitId, token, txOptions) {
```

Comment:
I think changing from `processExits` to `processExit` is dumbing it down too much. You may _want_ to process more than one exit, but now you can't 😢 

Also, changing the order of `token` and `exitId` is kind of confusing.